react/components/AutocompleteInput/README.md

```diff
@@ -39,6 +39,7 @@ const UsersAutocomplete = () => {
             ? user.toLowerCase().includes(term.toLowerCase())
             : user.label.toLowerCase().includes(term.toLowerCase())
         ),
+    size: 'small',
```

Comment:
Can you add an isolate example of sizes to this component?

Examples:
![Screenshot from 2020-06-18 17-49-27](https://user-images.githubusercontent.com/6867958/85070431-299e4f00-b18c-11ea-8c05-e022d3a4464a.png)
![Screenshot from 2020-06-18 17-49-54](https://user-images.githubusercontent.com/6867958/85070446-30c55d00-b18c-11ea-9910-a2adaba614bc.png)